plugins/modules/dcnm_vpc_pair.py

```diff
@@ -1104,9 +1103,6 @@ def dcnm_vpc_pair_translate_playbook_info(self, config, ip_sn, hn_sn):
             # Add other translations as required
             dcnm_vpc_pair_utils_translate_config(self, cfg)
 
-            # Check if the switches included in the config are Manageable.
-            dcnm_vpc_pair_utils_validate_devices(self, cfg)
```

Comment:
This is the only place in the DCNM collection that dcnm_vpc_pair_utils_validate_devices() is used.

If we don't anticipate using this function in the future, was wondering if we should also remove dcnm_vpc_pair_utils_validate_devices() from module_utils/network/dcnm/dcnm_vpc_pair_utils.py?

And if dcnm_vpc_pair_utils_validate_devices() is removed, then dcnm_vpc_pair_utils_check_if_meta() in the same file can also be removed since it's accessed only from dcnm_vpc_pair_utils_validate_devices()

Reply:
Yes I agree, I will remove it